core/pom.xml

```diff
@@ -80,6 +80,7 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
+            <version>2.6.0-SNAPSHOT</version>
```

Comment:
Should not this come from `<dependencyManagement>` in parent pom?

Reply:
this is to be removed, it's built on top of a PR not an actual branch. I wasn't thinking about how this only works for me because I populated my local maven cache with it.

Reply:
removed, note build will not succeed until AK changes are propagated to ccs